src/translators/crate.py

```diff
@@ -204,7 +206,7 @@ def _insert_entities_of_type(self,
                 msg = "Translating entity without TIME_INDEX. " \
                       "It should have been inserted by the 'Reporter'. {}"
                 warnings.warn(msg.format(e))
-                now_iso = datetime.now().isoformat(timespec='milliseconds')
+                now_iso = datetime.utcnow().isoformat(timespec='milliseconds')
```

Comment:
@c0c0n3 this ensure that also when not timeindex is inserted, we use utc

Reply:
excellent thanks!